tcms/rpc/tests/test_testplan.py

```diff
@@ -226,3 +222,114 @@ def test_ignores_existing_mappings(self):
                         case=case_id
                     ).count()
                 )
+
+
+@override_settings(LANGUAGE_CODE='en')
+class TestCreate(APITestCase):
```

Comment:
Missing scenario where you send `author_id` explicitly and it is different from the current user.